temporal-sdk/src/main/java/io/temporal/failure/ApplicationFailure.java

```diff
@@ -178,45 +188,7 @@ public static ApplicationFailure newNonRetryableFailureWithCause(
         ApplicationErrorCategory.UNSPECIFIED);
   }
 
-  /**
-   * New ApplicationFailure with a specified category and {@link #isNonRetryable()} flag set to
-   * false.
-   *
-   * <p>Note that this exception still may not be retried by the service if its type is included in
-   * the doNotRetry property of the correspondent retry policy.
-   *
-   * @param message optional error message
-   * @param type error type
-   * @param category the category of the application failure.
-   * @param cause failure cause. Each element of the cause chain will be converted to
-   *     ApplicationFailure for network transmission across network if it doesn't extend {@link
-   *     TemporalFailure}
-   * @param details optional details about the failure. They are serialized using the same approach
-   *     as arguments and results.
-   */
-  public static ApplicationFailure newFailureWithCategory(
```

Comment:
I thought this would be a breaking change but this isn't actually released, for any other readers.